src/main/java/com/snowflake/snowpark_java/Functions.java

```diff
@@ -3880,6 +3881,42 @@ public static Column listagg(Column col) {
     return new Column(com.snowflake.snowpark.functions.listagg(col.toScalaColumn()));
   }
 
+  /**
+   * Function to convert column name into column and order in descending manner.
```

Comment:
maybe use the same descriptin as here:
https://docs.snowflake.com/en/developer-guide/snowpark/reference/python/latest/snowpark/api/snowflake.snowpark.functions.desc
Also you can add an example